src/ufw.yml

```diff
@@ -6,36 +6,46 @@
   roles:
     - ufw
   vars:
-    # Do not actually enforce firewall rules.  The assessors will turn
-    # UFW on if they want it.
-    state: disabled
+    # Enable ufw and enforce firewall rules by default.
+    state: enabled
 
 - hosts: all
   name: Basic configuration for UFW
   become: yes
   become_method: sudo
   tasks:
-    - name: Allow incoming traffic through default TCP ports
+    - name: Allow incoming traffic through specified TCP ports
       community.general.ufw:
         proto: tcp
         rule: allow
         to_port: "{{ item }}"
       loop:
         - "22"       # ssh for Guacamole server (internal to assessment env)
         - "25"       # Mail (external)
-        - "53"       # C2 traffic (external)
-        - "80"       # Domain verification (external)
-        - "443"      # Payload hosting (external)
         - "587"      # Mail (external)
         - "993"      # IMAP (internal to assessment env)
+        - 5000:5999  # Additional listeners, if necessary (external)
         - "5901"     # vnc for Guacamole (internal to assessment env)
+        - "50050"    # CobaltStrike (internal to assessment env)
+    - name: Deny incoming traffic to specified TCP ports
+      # These rules may be manually modified by assessors as needed, but we
+      # want to default to a minimal set of allowed ports.
+      community.general.ufw:
+        proto: tcp
+        rule: deny
+        to_port: "{{ item }}"
+      loop:
+        - "53"       # C2 traffic (external)
+        - "80"       # Domain verification (external)
+        - "443"      # Payload hosting (external)
         - "8080"     # C2 listening and testing (external)
         - 8000:8999  # Additional listeners, if necessary (external)
-        - "50050"    # CobaltStrike (internal to assessment env)
-    - name: Allow incoming traffic through default UDP ports
+    - name: Deny incoming traffic to specified UDP ports
+      # These rules may be manually modified by assessors as needed, but we
+      # want to default to a minimal set of allowed ports.
       community.general.ufw:
         proto: udp
-        rule: allow
+        rule: deny
         to_port: "{{ item }}"
       loop:
         - "53"    # C2 traffic (external)
```

Comment:
This section can be removed as well.  The default behavior for incoming traffic is `deny`.

Reply:
I assume this is due to the same jazz as [here](https://github.com/cisagov/kali-packer/pull/92#discussion_r796747165).  Please add a comment, just as I requested [here](https://github.com/cisagov/kali-packer/pull/92#discussion_r796798197).

Reply:
Comment added in https://github.com/cisagov/teamserver-packer/pull/52/commits/16f2c947f9cb468a167a82a48892628484731c2d.